lib/openshift/cluster_service_version.rb

```diff
@@ -4,15 +4,12 @@ module BushSlicer
   class ClusterServiceVersion < ProjectResource
     RESOURCE = "clusterserviceversions"
 
-    # @return [BushSlicer::ResultHash] with :success depending on 
+    # @return [BushSlicer::ResultHash] with :success depending on
     # status phase=Succeeded and reason=InstallSucceeded;
-    def ready?(user: user, quiet: false, cached: false)
-    	res = get(user: user, quiet: quiet)
-    	if res[:success]
-    		res[:success] = 
-    			res[:parsed]["status"]["phase"] == "Succeeded" && res[:parsed]["status"]["reason"] == "InstallSucceeded"
-    	end
-    	return res
+    def ready?(user: nil, quiet: false, cached: false)
+      res = get(user: user, quiet: quiet)
+      res[:success] = res[:parsed]["status"]["phase"] == "Succeeded" && res[:parsed]["status"]["reason"] == "InstallSucceeded" if res[:success]
```

Comment:
Is there any change in logic? When `if` is at the end, it's not very visible. Can we keep using `if` on separate lines?

Reply:
no change in the logic, it was originally to fix the circular reference warning, but then I noticed the indentation issue (which only shows up in `cat`)

Reply:
@akostadinov anyway, I've put back the original formatting with the indentation fix